app/scripts/modules/core/src/managed/ManagedResourceStatusIndicator.tsx

```diff
@@ -102,8 +100,32 @@ const viewConfigurationByStatus = {
           <UISref to="home.applications.application.config" params={{ section: 'managed-resources' }}>
             <a>config view</a>
           </UISref>
-          . <LearnMoreLink id={id} />
+          .
+        </p>
+        <LearnMoreLink id={id} status={ManagedResourceStatus.PAUSED} />
+      </>
+    ),
+  },
+  RESUMED: {
+    iconClass: 'icon-md',
```

Comment:
@gcomstock FYI, right now I'm just using the unadorned "M" icon because 1) I don't have an icon for this new "resumed" status 2) this status theoretically will only show up for a few seconds at a time, so unlikely to matter much.

If you have time, might be cool to get a dedicated icon (maybe like a play icon to compliment the pause one?)

Reply:
Good idea! Here you are: https://github.com/spinnaker/deck/pull/7613

Reply:
EPIC thanks for the fast follow up